schedule/sles4sap_offline_gnome.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name:           sles4sap_offline_gnome
+description:    >
+  Graphical installation tests for SLES for SAP Applications without
+  SCC registration. Boot into Gnome.
+  Test SLES for SAP specific applications.
+vars:
+  APPTESTS: sles4sap/patterns,sles4sap/sapconf,sles4sap/saptune
+  DM_NEEDS_USERNAME: 1
+  HDDSIZEGB: 60
+  ROOTONLY: 1
+  SCC_REGISTER: never
+  SCC_URL:
+  SHUTDOWN_NEEDS_AUTH: 0
+  SLES4SAP_MODE: sles4sap
+  SLE_PRODUCT: sles4sap
+  SYSTEM_ROLE: default
+  WORKAROUND_MODULES: base,desktop,sapapp,serverapp,ha
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - sles4sap_product_installation_mode
```

Comment:
You forgot to add the correct path! So test is failing now... But easily fixed by this PR: #7538.

Reply:
Was this YaML file for `sles4sap_offline_gnome` generated automatically? If so, there is a problem with the generator. Worth taking a look at it.

Reply:
sorry, I missed the full path, It is generated manually because the final idea it is still on progress, what we mainly want to achieve next is to not change schedule for other teams.